app/api/server/v1/im.js

```diff
@@ -202,6 +202,12 @@ API.v1.addRoute(['dm.members', 'im.members'], { authRequired: true }, {
 
 		const { offset, count } = this.getPaginationItems();
 		const { sort } = this.parseJsonQuery();
+		const { status, username, name } = this.queryParams;
+
+		if (status && !Array.isArray(status)) {
+			throw new Meteor.Error('error-status-param-not-an-array', 'The parameter "status" should be an array of strings');
+		}
+
 		const cursor = Subscriptions.findByRoomId(findResult.room._id, {
```

Comment:
Add a projection to fetch only the `u` prop, since you're not using the rest of the result